dashboard/db/schema.rb

```diff
@@ -191,6 +191,20 @@
 
   add_index "games", ["intro_video_id"], name: "index_games_on_intro_video_id", using: :btree
 
+  create_table "hint_view_requests", force: :cascade do |t|
```

Comment:
In that case, I like user_id, (script_id, level_id) for the flexibility it gives us.